### PR TITLE
Bug fix 3.6/statistics short parameter only use ints

### PR DESCRIPTION
**Pull request by hkernbach**
Same as: https://github.com/arangodb/arangodb/pull/10932

### CODE REVIEW

**Comment by hkernbach** [comment]
http://jenkins01:8080/view/PR/job/arangodb-matrix-pr/8146/

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,10 @@
 v3.6.1 (2020-XX-XX)
 -------------------
 
+* Internal statistics API now uses integer timestamps instead of doubles. The
+  old behaviour sometimes leads to failed requests because of parse errors
```
Comment (jsteemann) [inline CHANGELOG]:
```suggestion
  old behavior sometimes leads to failed requests because of parse errors
```

File: CHANGELOG [hunk 2]
```diff
@@ -1,6 +1,10 @@
 v3.6.1 (2020-XX-XX)
 -------------------
 
+* Internal statistics API now uses integer timestamps instead of doubles. The
+  old behavior sometimes leads to failed requests because of parse errors
+	which occured in the internal used JavaScript joi library.
```
Comment (jsteemann) [inline CHANGELOG]:
```suggestion
	which occurred in the internally used JavaScript joi library.
```

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/8223/

**Comment by KVS85** [comment]
Tests blue.

**Pull request merged**